models/load_pearl_of_great_price.py

```diff
@@ -1,12 +1,53 @@
 """Load Pearl of Great Price."""
 
+import re
+
 from bs4 import BeautifulSoup
 from langchain.schema.document import Document
 
 from models.load_utils import clean
 from models.load_utils import to_markdown
 
 
+def remove_links(markdown_content: str) -> str:
+    """Remove links from the document."""
+    pattern = r"\[(.*?)\]\((.*?)\)"
+    return re.sub(pattern, "", markdown_content)
+
+
+def remove_lines_starting_with_pipe(text: str) -> str:
+    """Remove unwanted lines of text."""
+    # Pattern to match lines starting with "|"
+    pattern = r"^\|.*$"
+
+    # Use re.MULTILINE to match the pattern at the beginning of each line
+    result = re.sub(pattern, "", text, flags=re.MULTILINE)
+
+    return result
+
+
+def remove_non_texts(markdown_text: str) -> str:
```

Comment:
This removes things like `[the][scriptures]`. But I'm not sure that's necessary. Can you give me an example of what you want to remove with this pattern?